screens/NutritionScreen.js

```diff
@@ -1,23 +1,57 @@
-import React, { useState } from "react";
-import { StyleSheet, Text, SafeAreaView, View } from "react-native";
-import NutritionCard from "./Nutrition_Comp/NutritionCard";
-import TotalCaloriePercentage from "./Nutrition_Comp/TotalNutrition";
 import styles from "./styles.js";
+import firebase from "firebase";
+import React, { Component } from 'react';
+import {
+  SafeAreaView,
+  AppRegistry,
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+import Dates from 'react-native-dates';
+import moment, { now } from 'moment';
 
-const NutritionScreen = (props) => {
-  return (
-    <SafeAreaView style={styles.containerNutScreen}>
-      <View style={styles.percentageContainerNutScreen}>
-        <TotalCaloriePercentage />
-      </View>
+export default class NutritionScreen extends Component {
+  state = {
```

Comment:
Is it possible to change to react hooks?

Reply:
refactored to react hooks